### PR TITLE
Fixed error compiling in xcode 11 beta 5.

### PR DESCRIPTION
**Pull request by ykphuah**

### CODE REVIEW

**Comment by thii** [comment]
This causes a warning with Xcode 10.

**Comment by ruizmarc** [comment]
The warning is also present on Xcode 11, but without the fix of this pull request, it is impossible to compile for Xcode 11... 

In my opinion, I think it is better to have a warning than not being able to compile. Although I don't really like the idea of having a warning... it is better than nothing.

File: FontAwesome/Enum.swift [hunk 1]
```diff
@@ -2798,6 +2798,7 @@ public enum FontAwesome: String {
             case .youtube: return "\u{f167}"
             case .youtubeSquare: return "\u{f431}"
             case .zhihu: return "\u{f63f}"        
+            default: return ""
```
Comment (LukasHromadnik) [inline FontAwesome/Enum.swift]: This should be done in `codegen.swift` and not directly in the `Enum.swift` file

Comment (ykphuah) [reply FontAwesome/Enum.swift]: How do I run codegen.swift? It is failing at
```
guard let json = FileManager.default.contents(atPath: "FortAwesome/Font-Awesome/metadata/icons.json") else {
    fatalError("Could not find JSON metadata file")
}
```
I don't see the file icons.json inside the git repo.

Comment (LukasHromadnik) [reply FontAwesome/Enum.swift]: The file is in a submodule in `FortAwesome/FontAwesome`

Comment (ykphuah) [reply FontAwesome/Enum.swift]: I have updated codegen.swift, Enum.swift seems to be in the repo as well so I left the changes there.

**Comment by LukasHromadnik** [comment]
Changes look good to me. I'd prefer this PR to be merged even though there is a warning.

**Comment by romainbiard** [comment]
It's working well - big thank you 🙏 
When is it planned to be released ? 

For now we have in our PodFile:

>  pod 'FontAwesome.swift', :git => 'https://github.com/thii/FontAwesome.swift.git', :branch => 'master'
>   #pod 'FontAwesome.swift'

**Comment by thii** [comment]
This was released in 1.8.2.

**Comment by steventnorris-40AU** [comment]
Pod install cannot find release 1.8.2. Has the pod repo been updated with this release? Cannot build until this release is available.

**Pull request merged**